tests/data_analyzer/test_analysis.py

```diff
@@ -309,7 +309,7 @@ def test_check_for_new_twitches__returns_latest_twitch_index_and_populated_metri
 def test_DataAnalyzerProcess__sends_beta_1_metrics_of_all_wells_to_main_when_ready(
     four_board_analyzer_process, mantarray_mc_simulator, mocker
 ):
-    da_process, board_queues, from_main_queue, _, _ = four_board_analyzer_process
+    da_process, board_queues, from_main_queue, _, _, _ = four_board_analyzer_process
```

Comment:
Could use `da_process, board_queues, from_main_queue, *_`, but we really need to change the `four_board_analyzer_process` fixture to be a dict instead of tuple